viper.go

```diff
@@ -1373,7 +1373,7 @@ func readAsCSV(val string) ([]string, error) {
 
 // mostly copied from pflag's implementation of this operation here https://github.com/spf13/pflag/blob/master/string_to_string.go#L79
 // alterations are: errors are swallowed, map[string]interface{} is returned in order to enable cast.ToStringMap
-func stringToStringConv(val string) interface{} {
+func csvKeyValueConv(val string) interface{} {
```

Comment:
Wouldn't `stringToInterfaceCong` be a better name?

Reply:
Sure let me do that

Reply:
Done. But moved to another MR. Some issue happened in this MR
https://github.com/spf13/viper/pull/1462